packages/azure-storage-cli/README.md

```diff
@@ -0,0 +1,9 @@
+azure-storage-cli package
+============
+This package is used for communicating with azure storage account
+
+The blob compiled from [bosh-azure-storage-cli](https://github.com/cloudfoundry/bosh-azure-storage-cli)
```

Comment:
I know its the sam like in the S3 readme.md but shouldn't it at least be: `The blob *is* compiled from`